cl/cleanup/management/commands/fix_tax_court.py

```diff
@@ -107,6 +107,29 @@ def find_tax_court_citation(opinion_text):
                 return cite
 
 
+def fix_precedential_status(options):
+    """
+    This code corrects an on-going issues with scraping cases from tax court.
+
+    T.C. Memo(randum) cases are all being set to precedential when they should
+    not be. We can identify those cases by checking the citation type.
+
+    :param options:
+    :return: None
+    """
+    logger.info("Starting cleanup.")
+    ocs = OpinionCluster.objects.filter(
+        docket__court="tax", precedential_status="Published"
+    )
+    for oc in ocs:
+        if oc.citations.filter(type=Citation.NEUTRAL).exists():
+            logger.info("Updating cluster %s ", oc)
+            oc.precedential_status = "Unpublished"
+            oc.save()
```

Comment:
I think you should be able to do this whole thing with one query. Notionally:

    OpinionCluster.objects.filter(docket__court="tax", precedential_status="Published", citations__type=Citation.NEUTRAL).update(precedential_status="Unpublished")

The only problem with that is you also need to update solr, so if you do that, you should add a little code for capturing their IDs at some point and sending that along to Solr.

Reply:
Solr! Gets me everytime. I couldn't work our that one query though for the life of me.  I kept missing it somehow.

Reply:
Actually, never mind, the above is fine. It could be neater, but it'll work fine. Let's land it.